packages/react-components/react-alert/Spec.md

```diff
@@ -81,9 +99,11 @@ _Explain how the component will behave in use, including:_
   - _Touch_
   - _Screen readers_
 
+TODO - What goes here?
+
 ## Accessibility
 
-TODO -
+TODO - What goes here?
```

Comment:
Here goes any accessibility considerations you have to have for the component given the ARIA specifications.